client/src/components/History/Content/ContentItem.vue

```diff
@@ -169,6 +173,8 @@ export default {
         state() {
             if (this.isPlaceholder) {
                 return "placeholder";
+            } else if (this.item.element_count !== undefined && this.item.element_count === 0) {
```

Comment:
That is wrong, sorry. 0 elements is green.

Reply:
Ah I see, now I understand, I have also seen the comment on the other issue, now will look for a collection that is not "populated " instead

Reply:
Changed it so if we have a failed `populated_state`, we set the item to the newly created `failed_populated_state` state, which has the `danger` variant. Empty collections are still green